src/Kentico.Xperience.AzureSearch/Indexing/AzureSearchIndexClientService.cs

```diff
@@ -87,12 +97,14 @@ private async Task<SearchIndex> CreateIndexInternal(IList<SearchField>? searchFi
     }
 
 
-    private async Task<SearchIndex> EditIndexInternal(IAzureSearchIndexingStrategy strategy, string indexName, CancellationToken cancellationToken)
+    private async Task<SearchIndex> EditIndexInternal(IAzureSearchIndexingStrategy strategy, AzureSearchIndex azureSearchIndex, CancellationToken cancellationToken)
```

Comment:
Method is called from exactly 1 place where some check for index existence is performed. Is it necessary to check it there once again? If so, I believe that check can be combined.